src/style.css

```diff
@@ -0,0 +1,87 @@
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-weight: bold;
+}
+
+body {
+  margin-top: 0;
+  font-weight: 300;
+}
+
+[data-qa*="big"] {

```

Comment:
style classes

Reply:
it is not fixed. 
It's better to use classes instead of combine variant `.big input`. Check all over the document